src/click/types.py

```diff
@@ -768,6 +769,9 @@ class Path(ParamType):
         ``None``, keep Python's default, which is ``str``. Useful to
         convert to :class:`pathlib.Path`.
 
+    .. versionchanged:: 8.0.2
+        Added the `executable` parameter.
```

Comment:
Use double backticks to format code.

Reply:
Fixed, sorry for the lack of care with that.